src/ProjectTemplates/Web.ProjectTemplates/RazorComponentsWeb-CSharp.csproj.in

```diff
@@ -3,13 +3,10 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <NoDefaultLaunchSettingsFile Condition="'$(ExcludeLaunchSettings)' == 'True'">True</NoDefaultLaunchSettingsFile>
-    <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">RazorComponentsWeb_CSharp.Server</RootNamespace>
+    <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">RazorComponentsWeb_CSharp</RootNamespace>
+    <_RazorComponentInclude>Components\**\*.cshtml</_RazorComponentInclude>
```

Comment:
@rynowak I wasn't able to change the `_ViewImports.cshtml` to `_ViewImports.razor`, because they get ignored if I do. So this `_RazorComponentInclude` value is needed to make the remaining `.cshtml` files get treated as Razor Components.

Is this unexpected? Can we support `_ViewImports.razor` for preview 3?

Reply:
There is no support yet for `_ViewImports.razor` - I expected you would leave the file alone for now. What's the problem if you leave the file as `_ViewImports.cshtml` and remove `_RazorComponentInclude`?

Reply:
The `@layout` directive isn't valid unless it gets treated as a Razor Component:

```
1>Components\Pages\_ViewImports.cshtml(1,7,1,13): error CS0103: The name 'layout' does not exist in the current context
```

Reply:
OK, I guess we have to keep that workaround then 😢

Reply:
Yeah, I think the workaround is OK for preview 3. It's pretty well out of sight there in the `.csproj`.